kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDownloadPanel.js

```diff
@@ -109,6 +113,16 @@ define([
                             }
                         );
                     });
+                } else if (descr.name.toLocaleLowerCase() === 'json') {
+                    $dlBtn.click(() => {
+                        Jupyter.narrative.addAndPopulateApp(
+                            'kb_staging_exporter/export_json_to_staging',
+                            'dev',
```

Comment:
is this hardcoding the dev version of the app?

Reply:
The others are like that now, too, yeah. That'll get reworked in a future PR to use the current app tag.